### ISSUE TITLE
Staves not connected, octave shift line missing, words for finger guidance missing

### ISSUE THREAD

**Issue by bukaznik** [opensheetmusicdisplay/opensheetmusicdisplay#476]
Asking for a friend...

Hi this is Yolanda. I encountered some problems while dragging my xml file to OSMD demo. Could you plz offer me some help?

The attached is my xml file.

The problems are :

1. the measure group line for the two staves are not connected. 
2.the octave-shift line are missing part when the chrome are not full screen.
3.the words for the finger guidance while playing piano are missing.
4. further more, slurs are different with those on the picture. 

Don't know why😵 
really thx for your help.

xml: https://drive.google.com/file/d/1RZ6C9EY9W6M9eEdwclT7Rs4bLFMsOq3u/view?usp=sharing

![image](https://user-images.githubusercontent.com/14196717/50379148-a86f0600-0641-11e9-98f2-d269e539f9bc.png)

**Comment by bneumann** [comment]
Hi,

for your friend:

Had a look at the XML:

1. The instrument is defined as two parts so that's why the lines are not connected (as reference see the Clementi pieces vs. Beethoven in our examples). So this is due to the XML export or part definition.
2. Can't see that on my version: 
<img width="536" alt="image" src="https://user-images.githubusercontent.com/2605131/50384081-acda0400-06bf-11e9-8ed9-f130a58336f5.png">
3. Finger is not shown because it is not included in the XML. Again might be related to the creation tool or export settings.
4. (2.) The slurs are sometimes drawn beneath the lines instead above which might me due to the fact that they are 2 parts so the algorithm decides it is cool to draw it in between.

So basically: We can't show what's not there. Sorry lad :(
